host/4/powershell-build.yml

```diff
@@ -50,7 +50,7 @@ steps:
                   host/4/bullseye/amd64/powershell/powershell70
       npm run test $IMAGE_NAME --prefix  test/
       docker push $IMAGE_NAME
-    displayName: powershell70
+    displayName: powershell7
```

Comment:
I chose to change this to reflect the image name instead of the folder structure because I think that the image name is more descriptive of what is being produced.  Will gladly change it back if you would prefer 70.  I considered making labels be  powershell 7.0 but, that doesn't make much sense since nothing else is named that way

Reply:
Good point. Let's keep it as `PowerShell7`.